mu4/domain/notation/internal/notationplayback.cpp

```diff
@@ -62,9 +63,23 @@ NotationPlayback::NotationPlayback(IGetScore* getScore)
 {
 }
 
+void NotationPlayback::init()
+{
+    Ms::Score* score = m_getScore->masterScore();
+    IF_ASSERT_FAILED(score) {
+        return;
+    }
+
+    QObject::connect(score, &Ms::Score::posChanged, [this](Ms::POS pos, int tick) {
+        if (Ms::POS::CURRENT == pos) {
+            m_playPositionTickChanged.send(tick);
+        }
+    });
+}
+
 std::shared_ptr<MidiStream> NotationPlayback::midiStream() const
 {
-    Ms::Score* score = m_getScore->score();
+    Ms::Score* score = m_getScore->masterScore();
```

Comment:
It might be better to get rid of "master" term and call it just a `score()`

Reply:
I think it's best to keep the term "master". This is quite important, to understanding whether we are working with a master score or with an excerpt.